### ISSUE TITLE
In the Content Block Segmented Pattern the Supporting Link feature is missing

### ISSUE THREAD

**Issue by kexuibm** [carbon-design-system/ibm-dotcom-library#1541]
In both the story book as well as the react app for Content Block Segmented Pattern, the supporting links component does not seem to exist.

Please refer to the annotated visual design comp of the Learn Page linked below: 
https://ibm.box.com/s/y0zzgroneebz0blpa0l3q85myp4i10rt

If we look at the Complex Narrative Pattern, we can see a set of supporting links on the right of this pattern.

**Comment by jeffchew** [comment]
As discussed, we have opened issue https://github.com/carbon-design-system/ibm-dotcom-library/issues/1581 which would support the layout of the block patterns with the Link List.

**Comment by jeffchew** [comment]
@kexuibm , we have added in the new `Layout` option which should be available in the `v1.5.0-rc.0` release candidate for testing:

https://github.com/carbon-design-system/ibm-dotcom-library/tree/master/packages/react/src/patterns/sub-patterns/Layout

There is an additional type `2-1` which would be used, like so:

```
<Layout type="2-1">
    <div><ContentGroupSegmented /></div>
    <div><LinkList /></div>
</Layout>
```

Let us know if you run into any issues!

cc: @photodow @RaphaelAmadeu @ljcarot @RobertaJHahn
